packages/analytics/analytics-config-store/src/components/AnalyticsConfigCheck.vue

```diff
@@ -24,8 +24,8 @@ const analyticsConfigStore = useAnalyticsConfigStore()
 const analyticsConfig: Ref<ConfigStoreState> = analyticsConfigStore.getConfig()
 
 const loading = computed<boolean>(() => !analyticsConfig.value)
-const hasAnalytics = computed<boolean>(() => !!analyticsConfig.value && analyticsConfig.value.analytics)
-const hasPercentiles = computed<boolean>(() => !!analyticsConfig.value && analyticsConfig.value.analytics && analyticsConfig.value.percentiles)
+const hasAnalytics = computed<boolean>(() => !!analyticsConfig.value && !!analyticsConfig.value.analytics)
+const hasPercentiles = computed<boolean>(() => !!analyticsConfig.value && !!analyticsConfig.value.analytics?.percentiles)
```

Comment:
just a curiosity comment here, how come the first part of this isn't a redundant condition? 
could it not just be `!!analyticsConfig.value.analytics?.percentiles`?

Reply:
That was redundant -- thanks, good catch.  (I did run into a frustrating TS issue in `MetricsProviderInternal` where TS didn't like comparing `undefined` with a number, even though the outcome seemed stable.  But that's not the case here.)